frontend/src/App.js

```diff
@@ -28,6 +29,7 @@ function App() {
         
     </div>*/
     <Home/>
+    //<LandingPage/>
```

Comment:
Are these comments intentional?

Reply:
yes i did it intentionally. will remove them after testing